src/queues/ProtectedRoomActivityTracker.ts

```diff
@@ -55,6 +51,7 @@ export class ProtectedRoomActivityTracker {
      * Inform the tracker of a new event in a room, so that the internal ranking of rooms can be updated
      * @param roomId The room the new event is in.
      * @param event The new event.
+     *
      */
     public handleEvent(roomId: string, event: any): void {
```

Comment:
If `handleEvent` is meant to be called by some other `handleEvent`, maybe this should be documented?